packages/eas-cli/src/commands/branch/publish.ts

```diff
@@ -271,6 +311,10 @@ export default class BranchPublish extends Command {
       }
     }
 
+    if (!message && autoFlag) {
+      message = (await vcs.getLastCommitMessageAsync())?.trim();
```

Comment:
Similarly here, this fails when run on CI. It runs `git --no-pager log -1 --pretty=%B`, but since there's no git repo in an action, we'll see:

<img width="718" alt="Screen Shot 2021-08-06 at 12 15 09 AM" src="https://user-images.githubusercontent.com/6455018/128455170-81046921-920d-4620-aeb2-3b564d0b30e5.png">

Again, no need to update that logic in this PR, but let's make some tasks.